languages/zh-CN/SDK-PR-guidelines.md

```diff
@@ -0,0 +1,155 @@
+# 拉取请求指导
```

Comment:
It seems this translation already exists(/docs/SDK-maintainer-guidelines.md). Why do we have a redundant copy here?